taxonomies/ingredients.txt

```diff
@@ -17196,7 +17204,7 @@ ca:Oli de palmist
 cs:Palmojádrový olej
 da:Palmeolie
 de:Palmkernöl, Palmkern
-es:palmiste
+es:palmiste, aceite vegetal de coco
```

Comment:
It's different: palm kernel oil vs coconut oil

Reply:
oops - corrected